<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyComponentWidget</class>
 <widget class="QWidget" name="KeyComponentWidget">
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="addPage">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="addLabel">
         <property name="text">
          <string notr="true">Key Component: unset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string notr="true">Add Key Component</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="editPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="editLabel">
           <property name="text">
            <string notr="true">Change Key Component:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="componentWidgetLayout"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="laveUnchangedPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="componentSetLabel">
         <property name="text">
          <string notr="true">Key Component Set</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="changeButton">
           <property name="text">
            <string notr="true">Change Key Component</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeButton">
           <property name="text">
            <string notr="true">Remove Key Component</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
